src/entries/popup/pages/home/NFTs/NFTs.tsx

```diff
@@ -172,96 +168,185 @@ export function PostReleaseNFTs() {
-                  const section = sortedSections[index];
-                  const isLast = index === sortedSections.length - 1;
-                  return (
-                    <Box
-                      key={key}
-                      as={motion.div}
-                      position="absolute"
-                      width="full"
-                      style={{ height: size, y: start }}
-                      ref={collectionGalleryRowVirtualizer.measureElement}
-                      data-index={index}
-                    >
-                      <CollectionSection
-                        isLast={isLast}
-                        section={section}
-                        onAssetClick={onAssetClick}
-                      />
-                    </Box>
-                  );
-                })}
-            </Box>
-          </Box>
+          <>
+            {isInitialLoading ? (
+              <Box width="full">
+                <Inset horizontal="8px">
+                  <CollectionNFTsSkeleton />
+                </Inset>
+              </Box>
+            ) : (
+              <Box
+                width="full"
+                style={{
+                  height: collectionGalleryRowVirtualizer.getTotalSize(),
+                  minHeight: '436px',
+                  position: 'relative',
+                }}
+              >
+                <Box style={{ overflow: 'auto' }}>
+                  {collectionGalleryRowVirtualizer
+                    .getVirtualItems()
+                    .map((virtualItem) => {
+                      const { key, size, start, index } = virtualItem;
+                      const section = sortedSections[index];
+                      const isLast = index === sortedSections.length - 1;
+                      return (
+                        <Box
+                          key={key}
+                          as={motion.div}
+                          position="absolute"
+                          width="full"
+                          style={{ height: size, y: start }}
+                          ref={collectionGalleryRowVirtualizer.measureElement}
+                          data-index={index}
+                        >
+                          <CollectionSection
+                            isLast={isLast}
+                            section={section}
+                            onAssetClick={onAssetClick}
+                          />
+                        </Box>
+                      );
+                    })}
+                </Box>
+              </Box>
+            )}
+          </>
         )}
       </Box>
     </Bleed>
   );
 }
 
+function GroupedNFTsSkeleton() {
+  return (
+    <Box
+      style={{
+        display: 'flex',
+        flexWrap: 'wrap',
+        flexDirection: 'row',
+        justifyContent: 'flex-start',
+        gap: 16,
+        paddingBottom: 16,
+      }}
+    >
+      {Array.from({ length: 9 }).map((_, i) => {
```

Comment:
Idk if the 9 is also used on the actual UI (not skeleton) but if that's the case maybe we could export and reuse on both cases.

Same for the 16 that on the collections a few lines below